spec/dishes_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'dishes'
+describe Dishes do
+  let(:menu1) { Dishes.new ({ dish: 'Pizza', price: 5 }) }
```

Comment:
This line indicates to me that maybe your Dishes class would be better named Menu?

Reply:
I was also thinking that after the code review, thanks for the confirmation.